### DISCUSSION TITLE
SEB & DeepL

### DISCUSSION THREAD

**Discussion by NekoJonez** [SafeExamBrowser/seb-win-refactoring#810] [Q&A]
I have some students who are allowed to use DeepL... Now, if you would allow the app, the app has a history. Is it possible to clean that history when SEB starts or is that something the teachers/examiners need to look at?

**Comment by dbuechel** [comment]
If there is a command-line parameter you can pass to the app which would clean its history, then yes. Otherwise, that would indeed need to be handled by support personnel. Or you could allow the web application, e.g. via a URL filter.
